tutorials/avail-da.md

```diff
@@ -6,7 +6,52 @@ Avail DA offers scalable data availability that underpins the Avail ecosystem an
 
 This tutorial serves as a comprehensive guide for deploying your GM world rollup on Avail's data availability (DA) network.
 
-Before proceeding, ensure that you have completed the [GM world rollup](/tutorials/gm-world) tutorial, which covers setting up a local sovereign gm-world rollup and connecting it to a local (mock) DA node.
+Before proceeding, ensure that you have completed the GM World rollup setup. Follow the steps below to install Rollkit and GM World.
```

Comment:
I would revert this. Otherwise we are duplicating installation tutorial stuff. 

This tutorial should start with the assumption that rollkit is already installed, which is what this line was/is suggesting.